src/components/OptionsList.js

```diff
@@ -170,6 +175,7 @@ class OptionsList extends Component {
                 showSelectedState={this.props.canSelectMultipleOptions}
                 hideAdditionalOptionStates={this.props.hideAdditionalOptionStates}
                 forceTextUnreadStyle={this.props.forceTextUnreadStyle}
+                shouldShowFullDisplayNames={this.props.shouldShowFullDisplayNames}
```

Comment:
When would we not want to show full display names? Could we just always show them?

Reply:
Looks like there was a decision to display first names in https://github.com/Expensify/Expensify.cash/pull/2724. The OptionsList and its subcomponents are reused between the LHN and the Search Page. I wouldn't want to change what they did for that PR, so that's why I made it optional here.

Reply:
Ok, so then why are we making an exception here and showing the full names?

Reply:
Were they accidentally removed in that PR? I'm super confused.

Reply:
Doesn't look related, but maybe I'm missing something.

Reply:
Ok, I looked at the PR and I think I have a better idea about what the intention was with it. I'm not sure if we wanted the truncated names on the "Search" page. I also left some comments on that PR since I don't think it was handled correctly. Mind if we wait to see how it all pans out before adding this new prop in?

Reply:
Yeah absolutely!

Reply:
I'll stick these changes into a different PR.